networking_ccloud/tools/netbox_config_gen.py

```diff
@@ -370,6 +370,8 @@ def get_metagroups(self, connected_devices: Set[NbRecord]) -> List[conf.Hostgrou
         for name, count in Counter(cluster_names).items():
             if count > 1:
                 raise ConfigException(f'Cluster {name} appeared {count} times')
+        for name, metagroup in metagroups.items():
+            metagroup.members = list(sorted(metagroup.members))  # type: ignore
```

Comment:
You can use inplace sorting here, `metagroup.members.sort()`. Btw, sorted() should return a list already, not a generator or something else.